dlt/destinations/athena/athena.py

```diff
@@ -0,0 +1,253 @@
+            region_name=self.config.credentials.aws_region,
+            aws_access_key_id=self.config.credentials.aws_access_key_id,
+            aws_secret_access_key=self.config.credentials.aws_secret_access_key)
+        return self._conn
+
+    def close_connection(self) -> None:
+        self._conn.close()
+        self._conn = None
+
+    @property
+    def native_connection(self) -> Connection:
+        return self._conn
+
+    def create_dataset(self) -> None:
+        self.execute_sql(f"CREATE DATABASE {self.fully_qualified_dataset_name(escape=False)};")
+
+    def drop_dataset(self) -> None:
+        self.execute_sql(f"DROP DATABASE {self.fully_qualified_dataset_name(escape=False)} CASCADE;")
+
+    def fully_qualified_dataset_name(self, escape: bool = True) -> str:
+        return self.capabilities.escape_identifier(self.dataset_name) if escape else self.dataset_name
+
+    @contextmanager
+    @raise_database_error
+    def begin_transaction(self) -> Iterator[Any]:
+        yield self
+
+    @staticmethod
+    def _make_database_exception(ex: Exception) -> Exception:
+        if isinstance(ex, OperationalError):
+            if "TABLE_NOT_FOUND" in str(ex):
+                return DatabaseUndefinedRelation(ex)
+            return DatabaseTerminalException(ex)
+        elif isinstance(ex, (ProgrammingError, IntegrityError)):
+            return DatabaseTerminalException(ex)
+        if isinstance(ex, DatabaseError):
+            return DatabaseTransientException(ex)
+        return ex
+
+    def execute_sql(self, sql: AnyStr, *args: Any, **kwargs: Any) -> Optional[Sequence[Sequence[Any]]]:
+        with self.execute_query(sql, *args, **kwargs) as curr:
+            if curr.description is None:
+                return None
+            else:
+                f = curr.fetchall()
+                return f
+
+    @staticmethod
+    def _convert_to_old_pyformat(new_style_string: str, args: Tuple[Any, ...]) -> Tuple[str, Dict[str, Any]]:
+        # create a list of keys
+        keys = ["arg"+str(i) for i, _ in enumerate(args)]
+        # create an old style string and replace placeholders
+        old_style_string = re.sub(r"%s", lambda _: "%(" + keys.pop(0) + ")s", new_style_string)
+        # create a dictionary mapping keys to args
+        mapping = dict(zip(["arg"+str(i) for i, _ in enumerate(args)], args))
+        return old_style_string, mapping
+
+    @contextmanager
+    @raise_database_error
+    def execute_query(self, query: AnyStr, *args: Any, **kwargs: Any) -> Iterator[DBApiCursor]:
+        assert isinstance(query, str)
+
+        db_args = kwargs
+        # convert sql and params to PyFormat, as athena does not support anything else
+        if args:
+            query, db_args = self._convert_to_old_pyformat(query, args)
+            if kwargs:
+                db_args.update(kwargs)
+        try:
+            cursor = self._conn.cursor(formatter=DLTAthenaFormatter())
```

Comment:
also here you can split query by ";". that will OFC mess with parameters thing but in case of dict I expect you can just pass them all after splitting